lib/gpprmsya.gi

```diff
@@ -2447,6 +2447,42 @@ InstallMethod( RadicalGroup, "symmetric", true,
 InstallMethod( RadicalGroup, "alternating", true,
     [ IsNaturalAlternatingGroup and IsFinite], 0,RadicalSymmAlt);
 
+InstallMethod(NormalSubgroups,
+"for a perm group isomorphic to the symmetric group",
+[IsPermGroup and IsSymmetricGroup],
+function(G)
+  local pts, list;
+  pts := MovedPoints(G);
+  list := [Group(())];  # Trivial group
+  if Length(pts) = 4 then  # Klein 4-group
```

Comment:
This check is only valid if the symmetric group is in its "natural" form, as tested by `IsNaturalSymmetricGroup`. But  `Sym(4)` also has e.g. a permutation representation acting on 24 points (i.e. the regular module):
```
gap> G:=Image(RegularActionHomomorphism(SymmetricGroup(4)));;
gap> IsSymmetricGroup(G);
true
gap> IsNaturalSymmetricGroup(G);
false
gap> NrMovedPoints(G);
24
```